extensions/deployers/licenses_deploy.py

```diff
@@ -0,0 +1,11 @@
+from conan.tools.files import copy
+import os
+
+
+def deploy(graph, output_folder):
+    """Copies all the licenses from all the dependencies of the recipe"""
+    for name, dep in graph.root.conanfile.dependencies.items():
+        copy(graph.root.conanfile, "(COPY|COPYING|LICENSE).*",
```

Comment:
I remember other exotic names, like the license name itself and a typo, french file named as `LISENSE`. I suggest you running `find recipes -type f "conanfile.py -exec grep licenses {} +` to have more idea about license file names that we have in CCI

Reply:
Will make sure to get as many as possible once I add them in the new PR